exercises/linked-list/instructions.md

```diff
@@ -0,0 +1,13 @@
+# Instructions
+
+Your task is to write a prototype for managing train routes, represented as a doubly linked list.
+
+You don't need to worry about arrival and departure times at the stations for the prototype.
+Each station will simply be represented by a number.
```

Comment:
This might be a useful line for the simple-linked-list instructions too, linking the number to something from the story.